spec/pg_easy_replicate/orchestrate_spec.rb

```diff
@@ -681,4 +681,66 @@
       # described_class.restore_connections_on_source_db("cluster1")
     end
   end
-end
+  
+  describe ".exclude_tables" do
```

Comment:
`.exclude_tables ` convention signals that there is a function called `exclude_tables` in the file, since that is not the case, referencing application code with specs might get confusing. Perhaps best to just rename it 

```suggestion
  describe "excluding tables" do
```